core/src/main/scala/kafka/tools/ConsumerPerformance.scala

```diff
@@ -158,8 +158,22 @@ object ConsumerPerformance {
     val elapsedMs: Double = endMs - startMs
     val totalMBRead = (bytesRead * 1.0) / (1024 * 1024)
     val mbRead = ((bytesRead - lastBytesRead) * 1.0) / (1024 * 1024)
+    
```

Comment:
Could you please clarify the question? This method is called only when `currentTimeMillis - lastReportTime >= config.reportingInterval` (and we want a detailed report).